packages/client/components/GitLabScopingSearchResultsRoot.tsx

```diff
@@ -21,17 +21,26 @@ const GitLabScopingSearchResultsRoot = (props: Props) => {
         ...GitLabScopingSearchResults_meeting
         gitlabSearchQuery {
           queryString
+          selectedProjectsIds
         }
         teamId
       }
     `,
     meetingRef
   )
   const {teamId, gitlabSearchQuery} = meeting
-  const normalizedQueryString = gitlabSearchQuery.queryString.trim() || ''
+  const {queryString, selectedProjectsIds} = gitlabSearchQuery!
+  const normalizedQueryString = queryString.trim()
+  const normalizedSelectedProjectsIds = selectedProjectsIds?.length
+    ? (selectedProjectsIds as string[])
```

Comment:
+1 can remove casting, the end result can still be null

Reply:
I think we have to keep the casting here because if we remove it, TS complains that `normalizedSelectedProjectsIds` can be readonly